package.json

```diff
@@ -210,7 +210,8 @@
     "reason-textmate": "^2.1.0",
     "esy-sdl2": "*",
     "@opam/fmt": "^0.8.8",
-    "@opam/logs": "^0.7.0"
+    "@opam/logs": "^0.7.0",
+    "@opam/re": "*"
```

Comment:
Apparently `re` wasn't a dependency already, though it is an indirect dependency of something. Is it okay to add as direct dependency, or should we try something else?

Reply:
Should be fine to add as a direct dependency. If I recall correctly, we use it in revery anyway (so it gets compiled in regardless) 👍